src/AutoBalance.cpp

```diff
@@ -344,13 +349,83 @@ class AutoBalance_AllMapScript : public AllMapScript
         {
         }
 
+        void UnfairAuraImmunity(Map* map, Player* player, int enterExit) {
```

Comment:
the parameter enterExit should be a boolean instead, if/else is better than switch/case since the "default" case is never used. Otherwise please use a meaningful ENUM with an option or flag parameter instead.

Reply:
Understood